ui/app/css/itcss/components/newui-sections.scss

```diff
@@ -78,7 +78,6 @@ $sub-mid-size-breakpoint-range: "screen and (min-width: #{$break-large}) and (ma
 
   .main-container {
     width: 100%;
-    overflow-y: auto;
```

Comment:
What prompted this change?

Reply:
it shouldn't be there, I was switching between branches and trying to debug the issue with disappearing transaction history on firefox. Sorry @Gudahtt

Reply:
removed